detox/src/devices/drivers/android/genycloud/helpers/GenyAllocationHelper.js

```diff
@@ -0,0 +1,28 @@
+class GenyAllocationHelper {
+  constructor(instanceLookupService, instanceLifecycleService) {
+    this.instanceLookupService = instanceLookupService;
+    this.instanceLifecycleService = instanceLifecycleService;
+    this.coldBooted = undefined;
```

Comment:
What is the purpose of `this.coldBooted`? Does anyone from outside mutate this property?

Reply:
This code is quite outdated.